guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,47 @@
 {
```

Comment:
<p>* Fixes classcode predictions in the <code>predict_las()</code>  function for point cloud classification</p><p>* Fixes to <code>SuperResolution</code> model:</p><ul><li>Applies data augmentation to training data</li><li>Normalizes predicted color values in the range of (0,1)</li><li>Bug fixes to data preprocessing step when generating downsampled images</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>